spec/es6-extension.webidl

```diff
@@ -0,0 +1,13 @@
+// Extensions to the webidl.
+//
+// These extensions are not part of the AST spec, but of the compression mechanism.
+
+[ExtendsTypeSum=Expression]
+// An extension to `Expression` used to locally change probability tables.
+interface BinASTExpressionWithProbabilityTable {
```

Comment:
Can you add some more comment about the purpose of this, and how it's used?
also, will this be exposed to decoder?

Reply:
Ok, attempting to clarify.

In practice, the decoder will need to handle decompression of `BinASTExpressionWithProbabilityTable`. This should hopefully be a trivial rule.

Reply:
So, it's not implicitly resolved inside decompression, but exposed to token stream, right?

Reply:
Yes. Maybe we'll find a better way to do so in the future, but right now, the idea is to experiment with it.